pallets/confidential-asset/src/lib.rs

```diff
@@ -17,12 +17,27 @@
 //!
 //! The Confidential Asset module is one place to create the MERCAT security tokens on the Polymesh blockchain.
 
-use frame_support::{decl_error, decl_event, decl_module, decl_storage};
+use codec::{Decode, Encode};
+use cryptography::{
+    mercat::{
+        account::{convert_asset_ids, AccountValidator},
```

Comment:
Why is `AccountValidator` defined with `{}`?

Instead of the current
```rust
pub struct AccountValidator {}
```

you can have
```rust
pub struct AccountValidator;
```